pkg/autoscaler/autoscaler.go

```diff
@@ -262,21 +262,7 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		existingMachineSets, err := framework.GetMachineSets(client)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(existingMachineSets)).To(BeNumerically(">=", 1))
-
-		By("Getting existing machines")
-		existingMachines, err := framework.GetMachines(client)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(existingMachines)).To(BeNumerically(">=", 1))
-
-		By("Getting existing nodes")
-		existingNodes, err := framework.GetNodes(client)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(existingNodes)).To(BeNumerically(">=", 1))
```

Comment:
similar to above, this appears to be a sanity check but it is not directly about the scale up/down and the value is not used later.

Reply:
Same question.  Is this all nodes, or just a specific group of nodes?  Looks like it's probably ensuring we have at least one node, which I would assume is always true in our case.

Reply:
yeah, all nodes. my thought was similar, this is just checking to see that we have at least 1 node, but i don't think it adds to this specific test. if we want to ensure that we have a single node before beginning then i would move it to the out context.